test/org/javarosa/xform/parse/XFormParserTest.java

```diff
@@ -134,7 +134,7 @@ private void timeParsing(LargeInstanceFileGenerator lfg, Path largeDataFilename,
         NumberFormat nf = NumberFormat.getNumberInstance();
         List<String> results = new ArrayList<>(); // Collect and display at end
         results.add("Children\tSeconds");
-        for (double powerOfTen = 3; powerOfTen <= 6.0; powerOfTen += 0.1) {
+        for (double powerOfTen = 3; powerOfTen <= 4.0; powerOfTen += 0.1) {  // Raise this upper limit to really measure
```

Comment:
Shouldn't we document rough memory requirements? Seems like we know something.

Reply:
@dcbriccetti Yeah but not 4 GB I guess. So it's seems like JVM on CircleCI container could've pre-allocated too much and the process was killed. This would mean it's matter of CI config and we don't need this change. I guess. Anyway, I doesn't seem to be super important right now.

Reply:
This makes me think that that performance measurement code doesn’t belong in the tests. It certainly doesn’t need to run each time. Maybe I’ll find another place for it.

Reply:
Agreed they don't need to run every time.